src/test/resources/features/04_employment.feature

```diff
@@ -1503,14 +1155,14 @@ Feature: Employment Journeys - Income Tax Submission
 
   Scenario: Individual user has prior submission data, updates travel and entertainment to no, then does the full travel and entertainment flow
     When the user logs into the service with the following details
-      |Redirect url       |/2021/start                                          |
-      |Credential Strength|strong                                               |
-      |Confidence Level   |200                                                  |
-      |Affinity Group     |Individual                                           |
-      |Nino               |BB444444A                                            |
-      |Enrolment Key 1    |HMRC-MTD-IT                                          |
-      |Identifier Name 1  |MTDITID                                              |
-      |Identifier Value 1 |1234567890                                           |
+      | Redirect url        | /2021/start |
```

Comment:
Same as above comment, not sure why someones changing utilities to no aswell as travel. Lines 1172-1180 can be removed.

Reply:
Removed as suggested.